packages/catalog-process/src/model/domain/apiConverter.ts

```diff
@@ -205,3 +207,10 @@ export const eServiceToApiEService = (
   isConsumerDelegable: eservice.isConsumerDelegable,
   isClientAccessDelegable: eservice.isClientAccessDelegable,
 });
+
+export const apiEServiceTemplateVersionToEServiceTemplateVersion = (
```

Comment:
This method looks unused, and the new template fields for descriptor and eservice are never serialized in this file... I think something is missing. How are these fields retrieved/used by the BFF/API GW?